.github/workflows/govuln.yaml

```diff
@@ -16,4 +16,18 @@ jobs:
       - run: |
           set -euo pipefail
 
-          go install golang.org/x/vuln/cmd/govulncheck@latest && govulncheck ./...
+          go install golang.org/x/vuln/cmd/govulncheck@latest
+
+          # Find all go.mod files recursively, as we have multiple of them within the project
+          gomod_files=$(find . -name go.mod)
+
+          # Loop through each go.mod file path
+          for filepath in $gomod_files; do
```

Comment:
Usually looping over `find` results is not the best idea, see: https://www.shellcheck.net/wiki/SC2044

An alternative would be to do:

```sh
find -name go.mod -exec /bin/bash -c 'echo scanning $(dirname {}); govulncheck -C $(dirname {}) -show verbose ./...' \;
```

Reply:
Thank you for the suggestion! :) Patched as suggested!